tst/testinstall/zmodnz.tst

```diff
@@ -9,8 +9,36 @@
 ##
 gap> START_TEST("zmodnz.tst");
 
+#
+# ZmodnZ constructor
+#
+gap> ZmodnZ(4);
+(Integers mod 4)
+gap> ZmodnZ(-4);
+(Integers mod 4)
+gap> ZmodnZ(1);
+(Integers mod 1)
+gap> ZmodnZ(2);
+GF(2)
+gap> ZmodnZ(0);
+Integers
+gap> ZmodnZ(-1/2);
+Error, <n> must be an integer
+
+#
+# ZmodpZ constructor
+#
+gap> ZmodpZ(2);
+GF(2)
+gap> ZmodpZ(4);
+Error, <p> must be a prime
+
+#
 # small prime field
+#
 gap> Fam7:= ElementsFamily( FamilyObj( Integers mod 7 ) );;
+gap> z0:= ZmodnZObj( Fam7, 0 );
+ZmodpZObj( 0, 7 )
 gap> z1:= ZmodnZObj( Fam7, -3 );
 ZmodpZObj( 4, 7 )
```

Comment:
`ZmodpZObj` is not documented, and is this desirable?
```
gap> ZmodpZObj(0,4);
ZmodnZObj( 0, 4 )
```
(I realise that this might be far beyond saving because people rely on this behaviour)